doc/manual/manual.tex

```diff
@@ -8037,6 +8038,160 @@ \subsubsection{Melt migration in a 2D mantle convection model}
 A more complete comparison of the two models can be found in Section~4.7 ``Influence of melt migration on a global-scale 
 convection model'' in \cite{dannberg_melt}.
 
+\subsubsection{Melt migration in a 2D mid-ocean ridge model}
+\label{sec:cookbooks-mid-ocean-ridge}
+
+\textit{This section was contributed by Juliane Dannberg.}
+
+The following cookbook will explain how to set up a model of a mid-ocean ridge that uses \aspect{}'s 
+implementation of coupled magma/mantle dynamics (see Section~\ref{sec:melt_transport}) and melting 
+and freezing of mantle rock. 
+In particular, it will outline 
+\begin{enumerate}
+  \item how to use operator splitting to accurately compute melting and freezing of melt, 
+  \item how to use traction boundary conditions to set up the flow field of a mid-ocean ridge, 
+  \item useful strategies for how to refine the mesh in models with melt migration.
+\end{enumerate}
+How to set up a model with melt migration in general is explained in the previous cookbook \ref{sec:cookbooks-melt-global}. 
+
+As the flow at mid-ocean ridges can be assumed to be roughly symmetric with respect to the ridge axis 
+in the center, we only model one half of the ridge in a 2d Cartesian box with dimensions of $105 \times 70$\,km. Solid material is flowing in from the bottom with a prescribed temperature and melting due to decompression as is rises. The model is cooled from the top so that melt freezes again as it approaches this boundary. In addition, a fixed plate velocity away from the ridge axis is prescribed at the top boundary, inducing corner flow. Material can flow out freely at the right model boundary. The model shows both how melt is focused towards the ridge axis, and how melting and freezing induces chemical heterogeneity in the mantle, generating the crust and lithosphere. 
+A movie of the full model evolution can be found \href{https://www.youtube.com/watch?v=f4Bc4lzdNP0}{online}. 
+
+\paragraph{The input file.}
+One important problem in models with melting and freezing (and other reactions) is that these reactions 
+can be much faster than the time step of the model. For mid-ocean ridges, melt is generally assumed to 
+be in equilibrium with the solid, which means that the reaction is basically instantaneous. 
+To model these type of processes, \aspect{} uses operator splitting (see also Section \ref{sec:benchmark-operator_splitting}): Reactions are solved on a different time scale than advection. 
+For this model, this means that at the beginning of each time step, all melting reactions, 
+including their latent heat effects, are solved using a smaller time step. In the input file, 
```

Comment:
to avoid repeating "time step" twice here (with different meanings), maybe say something like

"this means that at the beginning of each time step, all melting reactions,  including their latent heat effects, are solved using several shorter sub-time steps"

or "...using several shorter time increments".